src/library/export/trackexport.cpp

```diff
@@ -0,0 +1,129 @@
+#include "library/export/trackexport.h"
+
+#include <QFileInfo>
+#include <QMessageBox>
+#include <QDebug>
+
+void TrackExport::run() {
+    int i = 0;
+    for (const auto& track : m_tracks) {
+        QString path = track->getLocation();
+        QFileInfo fileinfo(path);
+        emit(progress(fileinfo.fileName(), i, m_tracks.size()));
+        exportTrack(track);
+        if (m_bStop) {
```

Comment:
Now that it's a QAtomicInt it needs to use load_atomic -- see util/compatibility.h -- otherwise Qt5 build fails :-/.